doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -357,14 +357,19 @@ The device selector always has the form of ``DDDD:BB:DD.FF`` or ``DDDD.BB.DD.FF`
 
 and then set::
 
-  bluestore_block_path = spdk:0000:01:00.0
+  bluestore_block_path = "spdk:trtype:PCIe traddr:0000:01:00.0"
 
 Where ``0000:01:00.0`` is the device selector found in the output of ``lspci``
 command above.
 
+And you can also point NVMe Device provided by a remote NVMe-OF target with tcp
+transport via the following example::
+
+  bluestore_block_path = "spdk:trtype:TCP traddr:10.67.110.197 trsvcid:4420 subnqn:nqn.2019-02.io.spdk:cnode1"
+
```

Comment:
You may also specify a device provided by a remote NVMeoF target with the TCP transport as in the following example:

Reply:
ack